modules/outbreak_alerts/detail-layout.json

```diff
@@ -32,7 +32,7 @@
                                     "wid": "9a1e5385-5fb4-4c53-ab7a-c2c013fe5c99",
```

Comment:
module name rename from outbreak_alerts to outbreaks is not needed?

Reply:
not required